README.md

```diff
@@ -1,2 +1,170 @@
-# first-capstone-project
-My first capstone project at Microverse, finishing the Module 1
+# Capstone Project: Roland Garros 2023
+
+# ![Microverse bagde](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+> This is the first capstone project from microverse. this project is a website to promote the Roland Garros tournament, where people can know additional info about the tournaments and some featured players. It was built using HTML/CSS and Javascript
+
+## Screenshots
+
+- **Laptops/Desktops:**
+
+---
+# ![home_screenshot](./assets/images/capstone-desktop.jpg)
+
+
+- **Mobile:**
+
+---
+# ![Mobile screenshot](./assets/images/capstone-mobile.jpg)
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 HTML/CSS & JavaScript capstone project - Model: Conference page <a name="about-project"></a>
+
+**First capstone project** is a web page where you can find information about Rolland Garros Tournament.<br>
+
+In [this video](https://www.loom.com/share/3592df7602c04fd7ae8531b6eca35715) you can find more information about this project.
```

Comment:
- [ ] You did a great job presenting the project and implementing the required changes. :+1: I think you forgot to highlight a piece of code you feel most proud of. Please record the video again to include that detail. 🤝️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/235286658-f802eb41-1ab4-4ea7-8d0f-9899bc701e10.png)

Reply:
**I just recorded my video again**, sir, following your kindly suggestions.

**https://www.loom.com/share/4f041649025548649006fb8bf20d2b74**

**The Readme.md file was updated as well.**

Thanks, Mr. **@amejid**